### PR TITLE
Recommend Pantheon Secrets instead of Lockr

### LINKED ISSUE #9190: Using Lockr to Secure and Manage API and Encryption Keys Doc Update
Re: [Using Lockr to Secure and Manage API and Encryption Keys](https://docs.pantheon.iolockr)

Priority: Medium

## Issue Description:
Noting that our company was looking at LockR as a secure way to store our API keys but decided against it as their Wordpress plugin has not been supported/updated from the last 3 major releases.
https://wordpress.com/plugins/lockr

## Suggested Resolution
Better solutions for storing API keys securely in Wordpress.

### LINKED ISSUE #9200: WordPress Developer's Guide Should Steer Users to Secrets Management
Re: [WordPress Developer's Guide](https://docs.pantheon.io/guides/wordpress-developer/wordpress-secrets-management)

Priority: Medium

## Issue Description:

This doc still recommends deprecated (Lockr) or less-secure (`/files/private`) pathways for managing secrets. 

## Suggested Resolution

Update to point all users to Pantheon Secrets Management

### PR DESCRIPTION
**Pull request by ccharlton**
Fixes #9200 
Fixes #9190 

## Summary
Swapped out Lockr for Pantheon Secrets
Recommend Pantheon Secrets first, and private `secrets.json` file as an alternative 

* **[Integrations](https://pr-9247-documentation.appa.pantheon.site/integrations#security-and-authentication)** 
* **[Devleop](https://pr-9247-documentation.appa.pantheon.site/develop)** 
* **[Guides](https://pr-9247-documentation.appa.pantheon.site/guides)** 
* **[Automate](https://pr-9247-documentation.appa.pantheon.site/automate)** 
* **[Security](https://pr-9247-documentation.appa.pantheon.site/guides/security)** 
* [Secrets Management in WordPress](https://pr-9247-documentation.appa.pantheon.site/guides/wordpress-developer/wordpress-secrets-management#store-your-keys)
* [Private Paths for Code and Files](https://pr-9247-documentation.appa.pantheon.site/guides/secure-development/private-paths#private-path-for-files)
* [Quicksilver Install Scripts](https://pr-9247-documentation.appa.pantheon.site/guides/quicksilver/install-script#secrets)

I kept the [Lockr content](https://pr-9247-documentation.appa.pantheon.site/lockr) for now but added deprecated warning to the top and the title

### CODE REVIEW

**Comment by ccharlton** [comment]
I couldn't find how to edit: https://docs.pantheon.io/automate

**Review by rachelwhitton — APPROVED** [review]

**Comment by rachelwhitton** [comment]
Thanks for kicking this pr off @ccharlton! I did an audit and added every occurance of Lockr I found across the repo to this PR, if you have time / interest to review my changes please do!

@jazzsequence can you review the following as part of this PR: 

* [Secrets Management in WordPress](https://pr-9247-documentation.appa.pantheon.site/guides/wordpress-developer/wordpress-secrets-management#store-your-keys) 
* [Private Paths for Code and Files](https://pr-9247-documentation.appa.pantheon.site/guides/secure-development/private-paths#private-path-for-files)
* [Quicksilver Install Scripts](https://pr-9247-documentation.appa.pantheon.site/guides/quicksilver/install-script#secrets)

I kept the [Lockr content](https://pr-9247-documentation.appa.pantheon.site/lockr) for now but added deprecated warning to the top and the title

File: source/content/guides/wordpress-developer/11-wordpress-secrets-management.md [hunk 1]
```diff
@@ -32,43 +32,26 @@ Plugin and theme license keys are unique codes used to activate and authenticate
...
+
+Set the key via Terminus, then use the `pantheon_get_secret()` function in your script, for example:
 
 ```php
-if ( file_exists( dirname( __FILE__ ) . '/wp-content/uploads/private/wp-config-secrets.php' ) && isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ) {
-  if (!defined('WP_SAMPLE_KEY')) {
-    define( 'WP_SAMPLE_KEY', 'EXAMPLEKEY1234' );
-  }
+if ( function_exists('pantheon_get_secret') ) {
+  $secret_value = pantheon_get_secret("SECRET_NAME");
```
Comment (jazzsequence) [inline source/content/guides/wordpress-developer/11-wordpress-secrets-management.md:48]:
```suggestion
  $secret_value = pantheon_get_secret( 'secret_name' );
```

File: source/content/guides/quicksilver/02-install-script.md [hunk 2]
```diff
@@ -99,18 +99,16 @@ Use the [Pantheon Search and Replace URLs on WordPress Sites repository](https:/
...
-
-Install the [Terminus Secrets Plugin](https://github.com/pantheon-systems/terminus-secrets-plugin) to manage secret data in JSON files in your site's private files path.
-
-This allows you to write and remove key values. The code example below shows you how to write a value to a key:
-
-```bash{promptUser: user}
-terminus secrets:set site.env key value
+```php
+if ( function_exists('pantheon_get_secret') ) {
+  $secret_value = pantheon_get_secret("SECRET_NAME");
```
Comment (jazzsequence) [inline source/content/guides/quicksilver/02-install-script.md:106]:
```suggestion
if ( function_exists( 'pantheon_get_secret' ) ) {
  $secret_value = pantheon_get_secret( 'secret_name' );
```

**Comment by jazzsequence** [comment]
@rachelwhitton LGTM with the couple (minor) code style changes I suggested to the `pantheon_get_secrets` code blocks.

**Pull request merged**